src/config.rs

```diff
@@ -43,11 +42,12 @@ pub fn load_from(path: impl AsRef<Path>) -> anyhow::Result<Config> {
 }
 
 pub fn installed_config() -> Option<PathBuf> {
-    dirs::home_dir().map(|home| home.join(".gurk.toml"))
+    dirs::config_dir().map(|home| home.join("gurk/gurk.toml"))
```

Comment:
Could you also please rename the `home` variable? Also below. Thanks!

Reply:
done